how-to/spdockit-insights/search-alerts.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Search Alerts
+description: How to use alerts
+author: Vinko Bedek
+date: 22/5/2017
+---
+<!--
+1. treba napomenuti za licence, ali nisam siguran kak tocno 
+-->
+SPDocKit Insights provides the ability to be notified when a search query is satisfied. This is done as the events are being collected from the servers. When defining alerts, you can target a single farm, or apply the alert to all of the farms contained within the SPDocKit database. Once sent, the alert for a farm is not evaluated again for a configurable back off period. Without the back off period, our mailboxes would suffer greatly once a problem arises.
```

Comment:
_is not evaluated again for a configurable back off period._ - also sounds clunky. Maybe something along the lines:
_Once sent, the alert for a farm will not be evaluated again within a certain back off time period, which you can configure._
Not sure that doesn't sound clunky as well.